### DISCUSSION TITLE
Unable to get second Slope for Woehler curve

### DISCUSSION THREAD

**Discussion by NagelsThomas** [boschresearch/pylife#19] [Q&A] [answered]
Dear PyLife Community,

I am following the tutorial in the docs, there I am able to replicate all the plots except for the K_2 where it gets that second slant downwards.
Although K_2 does update in my woehler_curve_data, it does not show in the plots.
Is there an additional input which I am missing?

Thank you in advance!

**Comment by johannes-mueller** [comment, marked as answer]
Welcome @NagelsThomas 

I assume you are trying to replicate the plots from [this](https://pylife.readthedocs.io/en/latest/demos/woehler_curve.html#The-second-slope-k_2) section.

Can you please try to run the [corresponding demo notebook](https://github.com/boschresearch/pylife/blob/develop/demos/woehler_curve.ipynb) in a jupyter server. I've just tried it on my machine and there does not seem to be an issue.

Note that the key for the slope variable is `k_2` not `K_2`.
